homeworks/Vladimir.Karbivnychiy_VladimirSkarb/6-typescript/src/app.ts

```diff
@@ -0,0 +1,108 @@
+import './styles/style.css';
+
+const scoreElement = document.querySelector('[data-score]') as HTMLHeadingElement;
+const cubeElement = document.querySelector('[data-cube]') as HTMLDivElement;
+const cubeScoreElement = document.querySelector('[data-cube-score]') as HTMLDivElement;
+const keyElement = document.querySelector('[data-key]') as HTMLDivElement;
+
+class Game {
+    private score: number = 100;
+    private currentKey: string = '';
+    private interval: number = 2000;
+    private randomNumber: number;
+    private timer: NodeJS.Timeout;
+
+    constructor(
+        private scoreElement: HTMLHeadingElement,
+        private cubeScoreElement: HTMLDivElement,
+        private keyElement: HTMLDivElement,
+        private cubeElement?: HTMLDivElement,
+    ) {
+    }
+
+    start(): void {
+        this.startKeysInterval();
+        this.subscribeOnKeyPress();
+    }
+
+    private startKeysInterval(): void {
+        this.scoreElement.innerText = String(this.score);
+        this.timer = setInterval(() => {
+            this.setKey();
+        }, this.interval);
+    }
+
+    private setKey(): string {
+        const letter: string = 'abcdefghijklmnopqrstuvwxyz';
+        this.currentKey = letter.charAt(Math.floor(Math.random() * letter.length));
+        this.keyElement.innerText = this.currentKey.toUpperCase();
```

Comment:
This code repeats 3 times. It's better to create a method that updates the key element